lib/unhangout-sockets.js

```diff
@@ -169,17 +169,20 @@ _.extend(UnhangoutSocketManager.prototype, events.EventEmitter.prototype, {
 
         mgr.on("create-session", _.bind(function(socket, args) {
             var event = this.getEvent(args.roomId);
-            if (event && socket.user && socket.user.isAdminOf(event)) {
+            if (event && socket.user) {
                 var joinCap = parseInt(args.joinCap ||
                                        models.ServerSession.prototype.MAX_ATTENDEES);
                 if (isNaN(joinCap) || joinCap < 2 || joinCap > 10) {
                     return mgr.writeErr(socket, "Invalid joinCap value");
                 }
+
                 var newSession = new models.ServerSession({
                     title: args.title,
+                    proposedBy: socket.user,
```

Comment:
This is a privacy problem and a denormalization.  ``proposedBy`` will contain all user data, including email addresses, even if the person has chosen not to reveal their email address.  And it will also not get updated if the user changes their profile (e.g. updates picture, name, etc).

One solution is to only set the userId here.  Then we just have to make sure that the needed fields (e.g. picture, name, etc.) get included in the data shipped to the client with sessions.

Another solution is to use ``socket.user.toClientJSON``, which filters out fields that we consider private.  That would still denormalize the user data, but would avoid the privacy problems.